src/Pos/Ssc/NistBeacon.hs

```diff
@@ -68,6 +67,9 @@ instance SscGStateClass SscNistBeacon where
     sscLoadGlobalState = pure ()
     sscRollbackU _ = pure ()
     sscVerifyAndApplyBlocks _ _ = pass
-    sscCalculateSeedQ =
-        pure . Right . coerce . ByteArray.convert @_ @ByteString .
-            Hash.hashlazy @SHA256 . encode
+    sscCalculateSeedQ i _ = do
+        let h :: ByteString
+            h = ByteArray.convert $ Hash.hashlazy @SHA256 (encode i)
+        -- TODO: not sure that we don't have to cut it
```

Comment:
Cut what? Seed? Make it shorter? It should be 32 bytes here and all seeds are 32 bytes (maybe except genesis one).